src/asmcnc/apps/maintenance_app/widget_maintenance_spindle_settings.py

```diff
@@ -478,3 +478,7 @@ def update_font_size(self, value):
             value.font_size = 0.0275 * Window.width
         else:
             value.font_size = 0.03 * Window.width
+
+    def exit(self):
+        print("save spindle settings")
```

Comment:
Either use logger or remove if unnecessary

Reply:
Ah good catch, I was using it to validate the function gets called and forgot to remove it